spec/spec_helper.rb

```diff
@@ -1,37 +1,13 @@
-# See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
-
-ENV['RAILS_ENV'] ||= 'test'
+if ENV['CI']
+  require 'simplecov'
+  SimpleCov.start
+end
 
-# Load the ManageIQ environment
-require File.expand_path("../manageiq/config/environment", __FILE__)
```

Comment:
@Fryguy Note that this line literally is the conventional way you load the Rails environment, and takes the place of our homebaked version of doing so via loading the *app's* spec file (what we're talking about [here](https://github.com/ManageIQ/manageiq-graphql/pull/23/files#r161099467)). 

You want to keep this and just add it to the `rails_helper`.